### PR TITLE
Update return block - aws_region_info, autoscaling_group_info, cloudformation

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY

Updated return block of following modules
```
aws_region_info
autoscaling_group_info
cloudformation
```

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/autoscaling_group_info.py [hunk 1]
```diff
@@ -76,166 +76,272 @@
...
-    sample: ["Default"]
+    elements: dict
+    contains:
+        auto_scaling_group_arn:
+            description: The Amazon Resource Name of the ASG.
+            returned: success
+            type: str
+            sample: "arn:aws:autoscaling:us-west-2:123456789012:autoScalingGroup:10787c52-0bcb-427d-82ba-c8e4b008ed2e:autoScalingGroupName/dev11"
+        auto_scaling_group_name:
+            description: Name of autoscaling group
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:91]:
```suggestion
            description: Name of autoscaling group.
```

File: plugins/modules/autoscaling_group_info.py [hunk 2]
```diff
@@ -76,166 +76,272 @@
...
+                    },
+                    "lifecycle_state": "InService",
+                    "protected_from_scale_in": "false"
+                }
+            ]
+            contains:
+                availability_zone:
+                    description: The Availability Zone of the instance.
+                    type: str
+                    sample: ap-southeast-2a
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:167]:
```suggestion
                    sample: "ap-southeast-2a"
```

File: plugins/modules/autoscaling_group_info.py [hunk 3]
```diff
@@ -76,166 +76,272 @@
...
+                health_status:
+                    description: The last reported health status of the instance.
+                    type: str
+                    sample: "Healthy"
+                instance_id:
+                    description: The ID of the instance.
+                    type: str
+                    sample: i-012345678
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:175]:
```suggestion
                    sample: "i-012345678"
```

File: plugins/modules/autoscaling_group_info.py [hunk 4]
```diff
@@ -76,166 +76,272 @@
...
+                instance_type:
+                    description: The instance type size of the running instance.
+                    type: str
+                    sample: t2.micro
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:179]:
```suggestion
                    sample: "t2.micro"
```

File: plugins/modules/autoscaling_group_info.py [hunk 5]
```diff
@@ -76,166 +76,272 @@
...
+                    "LifecycleTransition": "autoscaling:EC2_INSTANCE_TERMINATING"
+                }
+            ]
+        load_balancer_names:
+            description: List of load balancers names attached to the ASG.
+            returned: success
+            type: list
+            sample: ["elb-webapp-prod"]
+        max_size:
+            description: Maximum size of group
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:239]:
```suggestion
            description: Maximum size of group.
```

File: plugins/modules/autoscaling_group_info.py [hunk 6]
```diff
@@ -76,166 +76,272 @@
...
+            returned: success
+            type: int
+            sample: 3
+        min_size:
+            description: Minimum size of group
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:244]:
```suggestion
            description: Minimum size of group.
```

File: plugins/modules/autoscaling_group_info.py [hunk 7]
```diff
@@ -76,166 +76,272 @@
...
+                {
+                    "key": "env",
+                    "value": "production",
+                    "resource_id": "public-webapp-production-1",
+                    "resource_type": "auto-scaling-group",
+                    "propagate_at_launch": "true"
+                }
+            ]
+        target_group_arns:
+            description: List of ARNs of the target groups that the ASG populates
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:305]:
```suggestion
            description: List of ARNs of the target groups that the ASG populates.
```

File: plugins/modules/autoscaling_group_info.py [hunk 8]
```diff
@@ -76,166 +76,272 @@
...
+            returned: success
+            type: list
+            sample: [
+                "arn:aws:elasticloadbalancing:ap-southeast-2:123456789012:targetgroup/target-group-host-hello/1a2b3c4d5e6f1a2b",
+                "arn:aws:elasticloadbalancing:ap-southeast-2:123456789012:targetgroup/target-group-path-world/abcd1234abcd1234"
+            ]
+        target_group_names:
+            description: List of names of the target groups that the ASG populates
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:313]:
```suggestion
            description: List of names of the target groups that the ASG populates.
```

File: plugins/modules/autoscaling_group_info.py [hunk 9]
```diff
@@ -76,166 +76,272 @@
...
+            returned: success
+            type: list
+            sample: [
+                "target-group-host-hello",
+                "target-group-path-world"
+            ]
+        termination_policies:
+            description: A list of termination policies for the group.
+            returned: success
+            type: str
```
Comment (alinabuzachis) [inline plugins/modules/autoscaling_group_info.py:323]: If it's a list, shouldn't type be list?

Comment (mandar242) [reply plugins/modules/autoscaling_group_info.py:323]: fixed. missed fixing that, was supposed to be
```
type: list
elements: str
```

**Comment by mandar242** [comment]
recheck

**Review by GomathiselviS — APPROVED** [review]

**Review by gravesm — APPROVED** [review]

**Comment by mandar242** [comment]
regate

**Review by alinabuzachis — APPROVED** [review]

**Comment by alinabuzachis** [comment]
regate

**Comment by mandar242** [comment]
regate

**Comment by mandar242** [comment]
regate

**Pull request merged**